### PR TITLE
Feature/ionic links

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add ionic links
- Remove react-native and ionic links from empty-state pattern

### CODE REVIEW

File: src/docs/patterns/appbar.mdx [hunk 1]
```diff
@@ -47,8 +47,6 @@ The basic app bar provides information about the current screen. Typically, this
             'https://pxblue-components.github.io/angular/?path=/story/components-dropdown-toolbar--with-full-config'
         }
         react={'https://pxblue-components.github.io/react/?path=/story/components-dropdown-toolbar--with-full-config'}
-        ionic
```
Comment (joebochill) [inline src/docs/patterns/appbar.mdx:50]: You could leave ionic here and point it to Angular if you want

**Review by joebochill — COMMENTED** [review]
Should we be removing / commenting out all of the RN examples for now since the import into Snack is not working?

**Comment by emclaug2** [comment]
> Should we be removing / commenting out all of the RN examples for now since the import into Snack is not working?

If our interactive examples are known to not work at this moment I'll comment them out.

File: src/docs/patterns/appbar.mdx (hunk 1 above)
Comment (emclaug2) [reply src/docs/patterns/appbar.mdx:50]: I think it's best to avoid pointing to the same resource.

**Comment by emclaug2** [comment]
All the RN interactive demo links are removed right now.

File: src/app/components/patterns/DemoCard.tsx [hunk 2]
```diff
@@ -127,13 +127,13 @@ export const DemoCard: React.FC<DemoCardProps> = (props): JSX.Element => {
                 {ionic && (
                     <DemoButton repository={repository} framework={'ionic'} url={ionic === true ? undefined : ionic} />
                 )}
-                {reactNative && (
+                {/*reactNative && (
```
Comment (joebochill) [inline src/app/components/patterns/DemoCard.tsx:130]: Only the react-native-design-patterns are not working. I'm not sure if we link out to anything else (component docs?) from a DemoCard, but you might want to double check that.

Comment (emclaug2) [reply src/app/components/patterns/DemoCard.tsx:130]: I looked around and didn't see any react-native resources that have demos, this fix should be good to go.

**Review by joebochill — APPROVED** [review]

**Pull request merged**